specifyweb/frontend/js_src/lib/components/Header/Notifications.tsx

```diff
@@ -142,7 +142,7 @@ export function Notifications({
           'aria-live': 'polite',
           className:
             unreadCount > 0 && !isOpen
-              ? '[&:not(:hover)]:!text-brand-300 [&:not(:hover)]:dark:!text-brand-400'
+              ? '[&:not(:hover)]:!text-brand-300 [&:not(:hover)]:dark:!text-brand-400 text-left'
```

Comment:
The issue affects not just this place, but every link in specify.
See a better fix in fc00e2b34 and e9d2d2ea9

Since it's already fixed on agent-merging branch, we can close this pull request